lib/active_model/serializer/array_serializer.rb

```diff
@@ -1,10 +1,11 @@
 module ActiveModel
   class Serializer
     class ArraySerializer
+      NoSerializerError = Class.new(StandardError)
       include Enumerable
       delegate :each, to: :@objects
 
-      attr_reader :meta, :meta_key
+      attr_reader :meta, :meta_key, :objects
```

Comment:
Do you want to keep `objects` reader around? It's no longer used. I could see an argument that it's a private interface.